mailpoet/lib/Mailer/WordPress/WordPressMailer.php

```diff
@@ -71,7 +73,7 @@ public function send() {
         $sendWithMailer($useFallback = true);
       } catch (\Exception $fallbackMailerException) {
         // throw exception passing the original (primary mailer) error
-        throw new \phpmailerException($e->getMessage(), $e->getCode(), $e);
+        throw new PHPMailerException($e->getMessage(), $e->getCode(), $e);
```

Comment:
These exception changes are because of the Namespace changes in WordPress core, right?

Reply:
This is because I changed the class `PHPMailerLoader`.